src/utilities/input.tsx

```diff
@@ -58,4 +95,68 @@ export const GetStatusIcon = (status: Validation, dataTestId: string): ReactElem
     }
     return null;
 };
+
 GetStatusIcon.displayName = 'FondueGetStatusIcon';
+
+export const InputActions = ({
+    clearable,
+    disabled,
+    obfuscated,
+    isObfuscated,
+    callbacks,
+    dataTestId = 'fondue-input-actions',
+}: InputActionsProps): ReactElement => {
+    return (
+        <span className="tw-flex tw-items-center tw-justify-between tw-w-auto">
+            {obfuscated && (
+                <Button
+                    disabled={disabled}
+                    size={ButtonSize.Small}
+                    style={ButtonStyle.Default}
+                    emphasis={ButtonEmphasis.Weak}
+                    onClick={callbacks.password}
+                    aria-label={`${isObfuscated ? 'show' : 'hide'} text input`}
```

Comment:
At some point, we will have to discuss internally those aria-label values. We enforce English language here and our internal buttons should support multi-language but I wouldn't worry about that for this specific PR but definitely a topic to review.

Reply:
Yes this is a good (and big) point.